packages/terra-alert/src/Alert.jsx

```diff
@@ -42,6 +43,10 @@ const propTypes = {
    * Adding `var(--my-app...` CSS variables is required for proper re-themeability when creating custom color styles _(see included examples)_.
    */
   customColorClass: PropTypes.string,
+  /**
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```

Comment:
We should add the @private syntax so this doesn't show up in the generated prop-types documentation.
```suggestion
   * @private
   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```